src/OrchardCore.Modules/OrchardCore.OpenId/Tasks/OpenIdBackgroundTask.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace OrchardCore.OpenId.Tasks
 {
-    [BackgroundTask(Group = "OrchardCore.OpenId")]
+    [BackgroundTask(Schedule = "* * * * *")]
```

Comment:
We may want to adopt a less aggressive value. What about every 15 or 30 minutes?

Reply:
Okay, just put 30 minutes